src/material/schematics/ng-update/migrations/theming-api-v12/migration.ts

```diff
@@ -40,13 +46,14 @@ export function migrateFileContent(content: string,
                                    oldMaterialPrefix: string,
                                    oldCdkPrefix: string,
                                    newMaterialImportPath: string,
-                                   newCdkImportPath: string): string {
+                                   newCdkImportPath: string,
+                                   extraSymbols: ExtraSymbols = {}): string {
```

Comment:
Should this be called `extraMaterialSymbols` given that it isn't passed into the CDK migration?